dev/com.ibm.ws.webcontainer.servlet.3.1.factories/src/com/ibm/ws/webcontainer31/osgi/webapp/factory/WebAppFactoryImpl31.java

```diff
@@ -21,22 +24,33 @@
 import com.ibm.ws.webcontainer31.osgi.webapp.WebApp31;
 import com.ibm.wsspi.injectionengine.ReferenceContext;
 
+import io.openliberty.checkpoint.spi.CheckpointHook;
+import io.openliberty.checkpoint.spi.CheckpointPhase;
+
 /**
  *
  */
-@Component(service = WebAppFactory.class, property = { "service.vendor=IBM" })
-public class WebAppFactoryImpl31 implements WebAppFactory {
+@Component(service = { WebAppFactory.class, CheckpointHook.class }, property = { "service.vendor=IBM" })
```

Comment:
No need for these to be CheckpointHooks if they are not going to override `prepare` or `restore` methods.